contracts/DepositManager.sol

```diff
@@ -171,41 +169,21 @@ contract DepositManager is DepositCore {
         }
     }
 
-    /**
-     * @notice Merges the deposit tree with the balance tree by
-     *        superimposing the deposit subtree on the balance tree
-     * @param _subTreeDepth Deposit tree depth or depth of subtree that is being deposited
-     * @param zero Merkle proof proving the node at which we are inserting the deposit subtree consists of all empty leaves
-     * @return Updates in-state merkle tree root
-     */
-    function finaliseDeposits(
```

Comment:
The verb finalise for deposits should be avoided because it implies nothing will ever change again. But chances are the submitted deposits might be enqueued back.

Reply:
Yeah that makes sense to me